### ISSUE TITLE
Anchordock title bars rendered in white after the first usage of the IDE

### ISSUE THREAD

**Issue by leoburn5** [zamtmn/metadarkstyle#31]
After installing the package, uppon atarting Lazarus, darkmode runs fine.

If I close IDE and start it again, the docked menu bars appears as white as showed belloe:

![image](https://github.com/zamtmn/metadarkstyle/assets/63558173/a23be6aa-aa7a-48c9-8b49-578e6f69b733)

If I unninstall the package and install again, at first usage, it´s fine, but the problem appears again in the subsequent uses.

**Comment by zamtmn** [comment]
Lazarus version? I have encountered such problems, they were introduced and quickly fixed https://gitlab.com/freepascal.org/lazarus/lazarus/-/issues/40375

**Comment by leoburn5** [comment]
> Lazarus version? I have encountered such problems, they were introduced and quickly fixed https://gitlab.com/freepascal.org/lazarus/lazarus/-/issues/40375

![image](https://github.com/zamtmn/metadarkstyle/assets/63558173/05e519c6-24d7-4822-b01c-804392c887fa)

![image](https://github.com/zamtmn/metadarkstyle/assets/63558173/b0c45624-6c10-4912-8f9f-e870098140c8)

**Comment by zamtmn** [comment]
Can`t confirm((
![image](https://github.com/zamtmn/metadarkstyle/assets/39580611/d697b329-26ce-484a-882e-5591d92d1ebe)

**Comment by leoburn5** [comment]
> Can`t confirm(( ![image](https://user-images.githubusercontent.com/39580611/264943821-d697b329-26ce-484a-882e-5591d92d1ebe.png)

You have to close IDE, and Open it Again.

The problem seem´s to be related with the dock option "Header Style" = Line or Lines.

I have changed to Themed Button, and it does not happens anymore.

This is a great feature, long waited by us Users, can this be made avaiable online package manager?

**Comment by zamtmn** [comment]
No, everything works for me((

> online package manager

Of course you can add, but I don't know how to do it. Can you help?

**Comment by leoburn5** [comment]
> No, everything works for me((
> 
> > online package manager
> 
> Of course you can add, but I don't know how to do it. Can you help?

Sure.

I’ll take a look into it!

**Comment by 0x4A4D00** [comment]
I encounter this problem in both versions 2.2.6 and Trunk. Launching Lazarus from the original binary file (Lazarus.exe without any shortcuts on the desktop) triggers the issue. However, when I start it with a shortcut pointing to startlazarus.exe or open it manually, it runs without problems. Furthermore, launching Lazarus.exe with the --no-splash-screen flag also resolves the issue.

![docked_problem](https://github.com/zamtmn/metadarkstyle/assets/40355105/0a11bbac-cecf-4417-a57f-7944d9233448)

**Comment by zamtmn** [comment]
Yes, confirm. I have this happening when starting Lazarus with a different settings folder

**Comment by theonlyasdk** [comment]
Does the problem go away when you disable `Fill Headers`?

**Comment by theonlyasdk** [comment]
Do you know where this initialisation has to be done?

**Comment by zamtmn** [comment]
Please check with latest lasarus and latest mds?

**Comment by theonlyasdk** [comment]
Have you made any changes, and do you want me to check it? Sorry, i don't have Lazarus installed currently and I'm not able to install it (storage issues). But I'll check it when I install Lazarus again. 👍

**Comment by zamtmn** [comment]
yes. changes:
mds - https://github.com/zamtmn/metadarkstyle/pull/86
lazarus - https://gitlab.com/freepascal.org/lazarus/lazarus/-/merge_requests/653

**Comment by theonlyasdk** [comment]
What does it fix? And does this mean dark mode is officially supported in Lazarus now? Anyway, it's an amazing change!
